Phonebook.Frontend/src/app/shared/components/navigation/navigation.component.html

```diff
@@ -1,177 +1,195 @@
+            <mat-icon>settings</mat-icon>
+            <span i18n="@@SettingsComponentTitle">Settings</span>
+          </button>
+        </mat-list-item>
+        <mat-list-item>
+          <button
+            mat-menu-item
+            [routerLink]="['/further-information']"
+            i18n-matTooltip="navigationBar|Info Button Tooltip@@navigationBarInfoTooltip"
+            matTooltip="Got Feedback? Than that`s the right place to click."
+            (click)="sidenav.close()"
+          >
+            <mat-icon>info_outline</mat-icon>
+            <span i18n="NavigationComponent|Link to Information page@@NavigationComponentInformation">Information</span>
+          </button>
+        </mat-list-item>
+        <mat-list-item>
+          <button
+            mat-menu-item
+            feedbackDrawer
+            i18n-matTooltip="navigationBar|Bug Button Tooltip@@navigationBarBugTooltip"
+            matTooltip="Found a Bug? Than that`s the right place to click."
+            (click)="sidenav.close()"
+          >
+            <mat-icon>bug_report</mat-icon>
+            <span i18n="NavigationComponent|Link to Bug page@@NavigationComponentBug">Report a Bug</span>
+          </button>
+        </mat-list-item>
+      </mat-list>
+    </nav>
+  </mat-sidenav>
+  <mat-sidenav-content>
+    <div class="navigation mat-elevation-z5">
+      <app-online-bar></app-online-bar>
+      <nav>
+        <mat-toolbar color="primary">
+          <mat-toolbar-row class="mat-elevation-z6">
+            <div class="left app-title">
+              <button mat-button [routerLink]="['/']">
+                <span
+                  class="mat-display-1"
+                  id="title"
+                  i18n="NavigationBar|Application Name@@navigationBarApplicationName"
+                  >Phonebook</span
+                >
+                <span
+                  class="beta-tag mat-headline pb-hide-below-small"
+                  *ngIf="currentEnvironment != Environment.production"
+                  >{{ getEnvironmentTag() }}</span
+                >
+                <span class="version pb-hide-below-small" *ngIf="currentEnvironment != Environment.development">
+                  v{{ version }}</span
+                >
+              </button>
+            </div>
+            <div id="pb-greetings" class="pb-hide-below-big" *ngIf="currentUser != null && hasImage">
+              {{ (getGreetingMessage() | async) + ' ' + currentUser.Firstname }}
+            </div>
+            <div class="pb-missing-userimage-info pb-hide-below-big" *ngIf="!hasImage">
+              <button
+                mat-stroked-button
+                class="pb-opacity-background mat-body-2"
+                (click)="navigateToOwnProfile()"
+                *ngIf="currentUser != null"
+                i18n-matTooltip="MissingUserImageMessage|Tooltip for MissingUserImage@@MissingUserImageTooltip"
```

Comment:
Please Provide the context of the Translation by using `NavigationComponent` instead of `MissingUserImageMessage`

Reply:
also with the other statements